introduction_to_applying_machine_learning/video_game_sales/video-game-sales-xgboost.ipynb

```diff
@@ -83,7 +60,12 @@
     "import sys                                        \n",
     "import math                                       \n",
     "import json\n",
-    "import boto3"
+    "import boto3\n",
+    "import sagemaker\n",
+    "\n",
+    "bucket = '<your-bucket-name>'  #  replace with bucket name in quotes only, NO URL path\n",
```

Comment:
change this line to be `bucket = sagemaker.Session().default_bucket()` - though you can mention that people can specify a bucket if they want too. You could also create a dedicated `sagemaker.Session()` object that gets passed around later in the notebook; some of our other notebooks do that.

Reply:
I thought about doing this, but unlike most of the sample notebooks, the data for this sample is hosted on Kaggle -- users need to fetch the data from Kaggle first and then manually put it in a bucket.  This last step is easier if it is a bucket the user named, rather than a SDK-generated bucket name which is hard to remember.